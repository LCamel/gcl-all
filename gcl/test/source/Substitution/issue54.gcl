var x, y, z : Int

{: P n = n > x + y :}
y := y + 1
x := x + y
{ P z }