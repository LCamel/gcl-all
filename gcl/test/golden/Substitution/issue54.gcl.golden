2: P [ x \ x + y ] z
  P [ x \ x + y ] z ===> z > x + y + y
    
3: P [ x \ x + y ] [ y \ y + 1 ] z
  P [ x \ x + y ] [ y \ y + 1 ] z ===> z > x + y + 1 + y + 1
    
5: P [ x \ x + y ] [ y \ y + 1 ] z
  P [ x \ x + y ] [ y \ y + 1 ] z ===> z > x + y + 1 + y + 1
    